index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav class="navbar" id="nav-bar">     
```

Comment:
- [ ] Kindly remove the HTML attributes that you are not using. Here you are not using the class attribute `class="navbar"` as a selector in your CSS. This will make your code cleaner and follow HTML and CSS best practices.

Reply:
✔️ Well done! This comment is direct, clear, and polite.